src/components/Header.js

```diff
@@ -41,6 +43,14 @@ export default function(store){
         }
       }
     },
+    mounted() {
```

Comment:
I feel like we probably don't want to run this repeatedly in an interval....what about a debounced scroll handler, so it is responsive to the user's interaction but also not too inefficient/blocking because it won't execute on every single scroll event?  Something like:

![image](https://cloud.githubusercontent.com/assets/1724544/25466197/b9473f3c-2abb-11e7-9654-e7699d34937b.png)

(I feel like it is probably ok to assume the header height is a fixed thing too rather than read it each time)